Mathlib/Algebra/Polynomial/Roots.lean

```diff
@@ -784,6 +824,27 @@ theorem map_roots_le_of_injective [IsDomain A] [IsDomain B] (p : A[X]) {f : A 
   · simp only [hp0, roots_zero, Multiset.map_zero, Polynomial.map_zero, le_rfl]
   exact map_roots_le ((Polynomial.map_ne_zero_iff hf).mpr hp0)
 
+theorem map_roots_card_le_degree {A B : Type*} [Semiring A] [CommRing B] [IsDomain B]
```

Comment:
```suggestion
theorem card_roots_map_le_degree {A B : Type*} [Semiring A] [CommRing B] [IsDomain B]
```
Because the LHS is `card (roots (map p f))` and the dot notation is merely notation. Same below.